### PR TITLE
feat: adds contact box section

### LINKED ISSUE #902: Modul for kontakt-oss
Oversikt over kontaktpunkter

Image

### PR DESCRIPTION
**Pull request by mikaelbr**
- Adds contact box section to Sanity in dark/light mode.
- Updates employee card to match with sizing and coloring.
- Extends Tag to support light/dark background and match design system in Figma.

See screenshots further down

Fixes https://github.com/varianter/variant.no/issues/902

<details><summary>screenshots</summary>
<p>

Screenshot 2024-11-30 at 21 37 17
Screenshot 2024-11-30 at 21 25 02
Screenshot 2024-11-30 at 21 24 53
Screenshot 2024-11-30 at 21 24 50
Screenshot 2024-11-30 at 21 24 46
Screenshot 2024-11-30 at 21 24 40
Screenshot 2024-11-30 at 21 24 37
Screenshot 2024-11-30 at 21 24 32

</p>
</details>

### CODE REVIEW

File: studio/schemas/objects/sections/contact-box.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+              validation: (rule) => rule.max(30).required(),
+              components: {
+                input: (props) =>
+                  StringInputWithCharacterCount({ ...props, maxCount: 30 }),
+              },
+            }),
+            defineField({
+              name: "email",
+              title: "Enter the email address for contact point",
+              type: "string",
```
Comment (idamand) [inline studio/schemas/objects/sections/contact-box.ts]: You can use type: "email", and get validation right out of the box 😊

File: studio/schemas/objects/sections/contact-box.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      options: {
+        list: backgroundOptions,
+        layout: "radio",
+      },
+      initialValue: ContactBoxBackground.Dark,
+    },
+    {
+      name: "contactPoints",
+      title: "List of people to contact",
+      description: "List of tags and people to contact within those tags.",
```
Comment (idamand) [inline studio/schemas/objects/sections/contact-box.ts:52]: What does `tags` mean in this case?

Comment (mikaelbr) [reply studio/schemas/objects/sections/contact-box.ts:52]: Tag in this case is the text used in buttons/tag to identify. Wanting to have it generic to support things like "Trondheim", "Oslo" or "Sales", "Recruitment" etc

File: studio/schemas/objects/sections/contact-box.ts (hunk 1 above)
Comment (mikaelbr) [reply studio/schemas/objects/sections/contact-box.ts]: Nice. Found it odd that I had to validate manually, but was just following a different scheme. But nice nice

File: studio/schemas/objects/sections/contact-box.ts (hunk 2 above)
Comment (idamand) [reply studio/schemas/objects/sections/contact-box.ts:52]: Nice! Perhaps add some additional explanation or an example to the description so it is easier to understand for the content creators

**Review by idamand — APPROVED** [review]

**Pull request merged**